core/conn/odb/src/odb.c

```diff
@@ -7080,7 +7080,7 @@ static void Oload(int eid)
             mfl = (size_t) etab[eid].td[i].Osize;
 
     /* Allocate field buffer */
-    if ( (str = (char *)malloc (mfl + 128)) == (void *)NULL ) {
+    if ( (str = (char *)calloc (1, etab[eid].buffsz + 1)) == (void *)NULL ) {
```

Comment:
Is the buffer str used to read in source data for string column and reused for the next row/column? I am anxious that we do not do this large allocation repeatedly. I feel this is addressed already, the question is more of a sanity check as I cannot trace the variable str back far enough.

Reply:
Hi Suresh
      str will only be allocated one time for each loading thread.